src/core_plugins/elasticsearch/index.js

```diff
@@ -15,8 +16,6 @@ const DEFAULT_REQUEST_HEADERS = [ 'authorization' ];
 
 module.exports = function ({ Plugin }) {
   return new Plugin({
-    require: ['kibana'],
```

Comment:
I gave this a try with c55b610.  I added the mappings class to kbnServer, it's not clear to me if that's the right place for it to live.  Thoughts?